doc/basics.md

```diff
@@ -53,6 +53,9 @@ rustup-toolchain-install-master -f -n master -c rustc-dev -c llvm-tools
 rustup override set master
 ```
 
+_Note:_ Sometimes you may get compiler errors when building Clippy, even if you
```

Comment:
This is a nit, but should I bold this and the other note lower down (so **Note:** instead of _Note:_) to match `CONTRIBUTING.md`?

Reply:
Normally we have _Note:_ in our docs. the note in the CONTRIBUTING.md file is an exception. So this is fine.,